src/test/java/unit/WorkspaceUser.java

```diff
@@ -154,14 +165,26 @@ private static void expectListedUserWithRoles(String userEmail, IamRole... roles
 
   /**
    * Helper method to call `terra workspace list` and filter the results on the specified user
-   * email.
+   * email. Uses the current workspace.
    */
-  private static Optional<UFWorkspaceUser> listWorkspaceUserWithEmail(String userEmail)
+  static Optional<UFWorkspaceUser> listWorkspaceUserWithEmail(String userEmail)
```

Comment:
maybe `getWorkspaceUserWithEmail` since it's not returning a list.

Reply:
The "list" here refers to the CLI command that's being called (`terra workspace list-users`), instead of the return value. Changed it to `workspaceListUsersWithEmail` instead. Open to other suggestions too.